recipes/gstreamer/all/conanfile.py

```diff
@@ -18,35 +26,28 @@ class GStreamerConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_introspection": [True, False],
+        "with_tools": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_introspection": False,
+        "with_tools": False,
     }
-    generators = "pkg_config"
+    generators = "PkgConfigDeps"
     _meson = None
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
     def requirements(self):
-        self.requires("glib/2.72.0")
-
-    @property
-    def _is_msvc(self):
-        return self.settings.compiler == "Visual Studio"
+        if self.version >= Version("1.20"):
+            self.requires("glib/2.74.0")
+        else:
+            self.requires("glib/2.72.0")
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
```

Comment:
```suggestion
        try:
            del self.settings.compiler.libcxx
        except Exception:
            pass
        try:
            del self.settings.compiler.cppstd
        except Exception:
            pass
```
rm_safe is Conan 1.52 which is not available in CCI yet.

Reply:
done